xml/art_raspberry-pi.xml

```diff
@@ -395,37 +309,27 @@
    &appliancename; is distributed as an XZ-compressed image file for &microsd;
    cards. This section will guide you through the process of preparing the card
    for the first boot. If you already have a &microsd; card containing the
-   image, you can skip this section and go straight to
+   image, you can skip this section and go to
    <xref linkend="sec-rpi-firstboot"/>.
   </para>
 
-  <note>
-   <title>SD card space requirements</title>
-   <para>
-    It is recommended to use a card with a capacity of at least 8&nbsp;GB.
-   </para>
-<!--<para>TO DO!!! 8 GB may lead to space issues with btrfs snapshots and
-        has not been tested for SP3. Suggestion: Leave as is and revisit
-        later.</para>-->
-  </note>
-
   <warning>
    <title>All data on the card will be lost!</title>
    <para>
     By following the procedure below, all data on the SD card will be
-    overwritten and therefore irrevocably lost. Be very careful when choosing
-    the destination device of the image writing process.
+    overwritten and therefore irrevocably lost. Pay particular attention when
+    choosing the destination device of the image writing process.
    </para>
   </warning>
 
   <sect2 xml:id="sec-rpi-installation-linux">
    <title>Preparing the card on Linux</title>
    <para>
-    Before and after you plug in the SD card, run the <command>lsblk</command>
-    command. Between the two runs of <command>lsblk</command>, there should be
-    a difference of one or more lines. The first column and first row is the
-    name of the node representing the SD card in your system. To write the
-    image to the card, use the <command>dd</command> command:
+    Insert the SD card and run the <command>lsblk</command>
+    command to identify the device path of the card (for example,
```

Comment:
I think the intention here was to identify on a desktop which of multiple devices is the right one to use - it may well be /dev/sdX devices, same as SATA disks (and multi-format USB adapters may give you 8+ at once). You appear to remove that here without explaining why? (no separate Git commits with description of changes)

Reply:
Basically, I find the original description potentially confusing. Expecting the user to count lines before and after, and them telling them to look for something like _the name of the node_ is not really helpful and prone to misunderstanding on the user's end. So I simplified the description. Also, if we don't expect the user to understand how to use the `lsblk` command, then we should definitely not ask them to use `dd` to create an image.